cmd/zrepl/zrepl.c

```diff
@@ -78,14 +95,24 @@ open_zvol(int fd, zvol_info_t **zinfopp)
 		hdr.status = ZVOL_OP_STATUS_FAILED;
 		goto open_reply;
 	}
+	if (zinfo->state != ZVOL_INFO_STATE_ONLINE) {
```

Comment:
Do we initialize `zinfo->state` to `ZVOL_INFO_STATE_ONLINE`?

Reply:
done